docs/pages/aws-terraform-guide.mdx

```diff
@@ -150,7 +150,7 @@ cluster from scratch, so choose carefully.
 
 Setting `export TF_VAR_ami_name="gravitational-teleport-ami-ent-(=teleport.version=)"`
 
-Gravitational automatically builds and publishes OSS, Enterprise and Enterprise FIPS 140-2 AMIs when we
+Gravitational automatically builds and publishes Community, Enterprise, and Enterprise FIPS 140-2 AMIs when we
```

Comment:
Teleport (our official company brand now) builds and publishes Open Source, Enterprsie, and ...